main/monmon.rb

```diff
@@ -1,10 +1,10 @@
 require 'csv'
-
+params = ARGV[1]
 balance = Hash.new(0)
 not_supported = Hash.new(0)
 SUPPORTED_CURRENCIES = %w[BYN USD RUR EUR].freeze
 
-CSV.foreach('input.csv', headers: true, header_converters: :symbol) do |row|
+CSV.foreach(params, headers: true, header_converters: :symbol) do |row|
```

Comment:
The algorithm will break. Ok. I use option parser.

Reply:
please, pay attention to the error. try it and see how it looks like. it's a very common exception in Ruby, and it takes time and practice to prevent these errors